R/constants.R

```diff
@@ -31,6 +32,14 @@ GTAP_HISTORICAL_YEAR <- 2000
 aglu.GLU <- "GLU"
 aglu.GLU_NAME_DELIMITER <- ""  # delimiter between the GLU name and number
 
+# cotton seed and lint conversion
+CONV_COTTON_LINT <- 0.4
+
+# Price conversion from alfalfa to grass hay
+PRICERATIO_GRASS_ALFALFA <- 0.7
```

Comment:
Here it is: http://www.fao.org/es/faodef/fdef06e.htm
"FAO considers cottonseed, cotton lint and linters to be primary products. Lint content ranges from 30 to 40%, seed 55 to 65%, and linters 2 to 5% though they are not always separated."